app/experimenter/legacy-ui/rapid/components/experiments/ResultsTable.tsx

```diff
@@ -0,0 +1,270 @@
+import React from "react";
+import ReactTooltip from "react-tooltip";
+
+import {
+  SIGNIFICANCE_TIPS,
+  RESULT_COLUMN_TIPS,
+} from "experimenter-rapid/components/experiments/tooltips";
+import { featureOptions } from "experimenter-rapid/components/forms/ExperimentFormOptions";
+import { AnalysisPoint, ExperimentData } from "experimenter-types/experiment";
+
+enum Significance {
+  POSITIVE,
+  NEGATIVE,
+  NEUTRAL,
+}
+
+// A mapping of metric label to relevant statistic. This is
+// used to see which statistic will be used for each metric.
+const resultsMetricsMap = {
+  retained: "binomial",
+  search_count: "mean",
+  identity: "count",
+};
```

Comment:
Is this mapping something that could be done at the data translation layer on the backend, ie in the view/lookup function?

Reply:
yes, I've updated to move this there